gift-certificate-web/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>GiftCertificateMaven</artifactId>
+        <groupId>com.epam.esm</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <spring.version>5.2.9.RELEASE</spring.version>
+        <jackson.version>2.11.3</jackson.version>
+        <thymeleaf.version>3.0.11.RELEASE</thymeleaf.version>
+        <log4j.version>2.13.2</log4j.version>
+    </properties>
+
+    <artifactId>gift-certificate-web</artifactId>
+
+    <dependencies>
+        <dependency>
+            <artifactId>gift-certificate-data</artifactId>
+            <groupId>com.epam.esm</groupId>
+            <version>1.0.0-SNAPSHOT</version>
+        </dependency>
+
+        <!-- Spring + Spring REST -->
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
This dependency is present in the parent module too. So, it would be great to remove duplicates.